ndmg/stats/qa_reg.py

```diff
@@ -213,16 +235,35 @@ def plot_overlays(atlas, b0, cmaps=None, minthr=2, maxthr=95, edge=False):
                 image = edge_map(image).data
                 image[image > 0] = max_val
                 image[image == 0] = min_val
-
+            #Set the axis invisible
+            plt.xticks([])
+            plt.yticks([])
+            
+            #Set the frame invisible
+            ax.spines['top'].set_visible(False)
+            ax.spines['right'].set_visible(False)
+            ax.spines['bottom'].set_visible(False)
+            ax.spines['left'].set_visible(False)
+            
             ax.imshow(atl, interpolation="none", cmap=cmaps[0], alpha=0.9)
             ax.imshow(
                 opaque_colorscale(
                     cmaps[1], image, alpha=0.9, vmin=min_val, vmax=max_val
                 )
             )
-
+            
+            #set the legend
+            if idx == 3:
+                plt.plot(0,0,"-",c="pink",label='registered')
+                plt.plot(0,0,"-",c="green",label='reference')
+                plt.legend(loc='best',fontsize=12,bbox_to_anchor=(1.5,1.5))
+            
+    #Set title for the whole picture
+    [a,b,c] = shap
+    title = 'QA For Registration\n\nVolume:'+ str(a) +'*'+ str(b) + '*' + str(c)+'\n'
+    foverlay.suptitle(title,fontsize=24)
     foverlay.set_size_inches(12.5, 10.5, forward=True)
-    foverlay.tight_layout()
+    foverlay.tight_layout
     return foverlay
 
 
```

Comment:
In general, I don't see very much (read: any) code here that I didn't see in the other PRs, and this is supposed to be one PR from two people.

Did you guys write `opaque_colorscale`, `plot_overlays`, or `pad_im`? If not, what code did you actually write?

Reply:
Let me explain. Opaque_colorscale and plot_overlays are all existed functions in reg_utils.py created by Eric Bridgeford on 2017-06-21 but somehow not used in the pipeline. We used them to generate qa for registration steps and fixed some of the details to meet Jovo's requirements. We did write our own qa codes but we think it's a more convenient way to fix the existed codes and the qa screens were really better than ours.

Reply:
After we fixed the original QA function, Chuankai and Jialin thought it was a good way to do qa. So they used similar functions as ours. pad_im is a work of Wilson